pkgs/games/osu-lazer/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -I nixpkgs=../../../. -i bash -p curl jq common-updater-scripts
```

Comment:
why?

Reply:
`maintainers/scripts/update.nix` script assumes it is run in the root of the `nixpkgs` repo.